tests/rules/test_new_lines.py

```diff
@@ -13,6 +13,9 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+from unittest import mock
+
+import yamllint
```

Comment:
Is this import useful? I guess you wrote it for *mock*, but I don't think it's required.